blocks/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+# Create your models here.
+
+
+class Block(models.Model):
+    '''
+    This represents a block in the blockchain
+    '''
+    hash = models.CharField(max_length=100)
```

Comment:
Hash is, usually mentioned in your diagram, a 160-bit character (SHA-1) and is 40 HEX characters long. Shouldn't that be stored in a 40 Character max_length?

Reply:
Since hash is immutable, make it editable=False

Reply:
The same goes for merkle_hash and previous_blocks. ^

Reply:
fixed